Script/AtomicEditor/ui/modal/build/BuildSettingsWindow.ts

```diff
@@ -41,7 +41,7 @@ export class BuildSettingsWindow extends ModalWindow {
         platformSource.addItem(new Atomic.UISelectItem("Windows", "WindowsBuildSettings", "LogoWindows"));
         platformSource.addItem(new Atomic.UISelectItem("Mac", "MacBuildSettings", "LogoMac"));
         platformSource.addItem(new Atomic.UISelectItem("Android", "AndroidBuildSettings", "LogoAndroid"));
-        platformSource.addItem(new Atomic.UISelectItem("iOS", "iOSBuildSettings", "LogoIOS"));
+        platformSource.addItem(new Atomic.UISelectItem("iOS", "iosBuildSettings", "LogoIOS"));
```

Comment:
iOS is kind of a wildcard, liking iosBuildSettings in terms of a "rule" I guess, could go either way, let's stick with this...